lib/generators/react_on_rails/templates/base/base/lib/tasks/assets.rake

```diff
@@ -19,8 +19,6 @@ namespace :assets do
   end
 
   task :clobber do
-    rm_rf "#{Rails.application.config.root}/app/assets/javascripts/generated/vendor-bundle.js"
-    rm_rf "#{Rails.application.config.root}/app/assets/javascripts/generated/client-bundle.js"
-    rm_rf "#{Rails.application.config.root}/app/assets/javascripts/generated/server-bundle.js"
+    rm_rf Dir.glob(Rails.root.join("app/assets/javascripts/generated/*"))
```

Comment:
@rupurt while we're at it, let's change to `rm_r` if that's available for Rake.

no need for rm -rf, right?